docker-compose-build.yml

```diff
@@ -16,12 +16,25 @@ services:
     image: postgres:9.6
     restart: always
     environment:
-      - POSTGRES_PASSWORD=postgres_pass
       - POSTGRES_USER=postgres_user
+      - POSTGRES_PASSWORD=postgres_pass
     volumes:
       - ./pg_data_build:/var/lib/postgresql/data
       - /etc/localtime:/etc/localtime:ro
 
+  pgbouncer:
```

Comment:
@jamilatta este serviço esta rodando como `root`? se for, deve ser trocado para um usuário com menos privilégios, o mesmo aplica para o serviço: `postgres`